test_project/test_utils/dbt_test_utils.py

```diff
@@ -291,15 +317,21 @@ def clean_target():
         shutil.rmtree(target, ignore_errors=True)
 
     @staticmethod
-    def clean_csv():
+    def clean_csv(model_name=None):
         """
         Deletes csv files in csv folder.
         """
 
-        delete_files = [file for file in glob.glob(str(CSV_DIR / '*.csv'), recursive=True)]
+        if not model_name:
+            delete_files = [file for file in glob.glob(str(CSV_DIR / '*.csv'), recursive=True)]
+        else:
+            delete_files = [Path(f"{CSV_DIR}/{model_name.lower()}.csv")]
```

Comment:
Can just be:

`CSV_DIR / f"{model_name.lower()}.csv"`

Reply:
No, this gives an error that a PosixPath iterable list is required, hence the [Path( ... )].
I have however swapped the condition around now so that it is if model_name: ... else:

Reply:
OK, I tried this again and indeed this works, code now changed. :-)